archivers/star/Portfile

```diff
@@ -1,51 +1,204 @@
+                        Schily's pax (or spax) archiver is fully compliant with the POSIX \
+                        1003.1-2001 standard!  In other words, unlike the pax \
+                        archiver provided by macOS, the Schily pax (or spax) archiver will \
+                        actually create and read archives in the POSIX 1003.1-2001 pax \
+                        archive format! \
+                        \n \
+                        \nAlso included is the Schily tartest utility used to test whether a \
+                        tar archive is POSIX compliant and the star_sym utility which \
+                        reads a inode database file from a previous incremental restore via \
+                        the star archiver and prints the database in a human readable format. \
+                        \n \
+                        \nOmitted from the port are the mt, smt and rmt binaries to control \
+                        magnetic tape drives.
 
-depends_build       port:smake
+checksums               rmd160  e154278ecbe7d778bc1d6766ed163c9963b1cc82 \
+                        sha256  a4270cdcca5dd69c0114079277b06e5efad260b0a099c9c09d31e16e99a23ff5 \
+                        size    5896292
 
-worksrcdir          star-${version}
+patch {
+                        reinplace -locale C "s|gnutar|sgnutar|g" \
+                        ${worksrcpath}/star/gnutar.1
 
-use_configure       no
+                        reinplace -locale C "s|-noclobber| |g" \
+                        ${worksrcpath}/cdrecord/Makefile.dfl
 
-# smake does not support the -j flag
-build.jobs          -1
+                        reinplace -locale C "s|/opt/schily|${prefix}|g" \
+                        ${worksrcpath}/DEFAULTS/Defaults.darwin \
+                        ${worksrcpath}/DEFAULTS/Defaults.mac-os10 \
+                        ${worksrcpath}/DEFAULTS_ENG/Defaults.darwin \
+                        ${worksrcpath}/DEFAULTS_ENG/Defaults.mac-os10 \
+                        ${worksrcpath}/libfind/find.c \
+                        ${worksrcpath}/libfind/find_main.c \
+                        ${worksrcpath}/librscg/scsi-remote.c \
+                        ${worksrcpath}/patch/tests/random/cmptest.sh \
+                        ${worksrcpath}/patch/tests/random/gentest.sh \
+                        ${worksrcpath}/sformat/datio.c \
+                        ${worksrcpath}/smake/job.c \
+                        ${worksrcpath}/smake/make.c \
+                        ${worksrcpath}/star/star.1 \
+                        ${worksrcpath}/sunpro/Make/bin/make/common/misc.cc \
+                        ${worksrcpath}/sunpro/Make/bin/make/common/read2.cc \
+                        ${worksrcpath}/TEMPLATES/Defaults.gcc \
+                        ${worksrcpath}/TEMPLATES/Defaults.clang \
+                        ${worksrcpath}/TEMPLATES/Defaults.xcc \
+                        ${worksrcpath}/ved/ved.h
+}
+
+conflicts_build-append  cdrtools
```

Comment:
I'm guessing that you'll update this to use the conflict-check proc, as with the other ports?

Reply:
Yes!  Somehow I forgot to add it in the last push.  

Your procedure and check has been added.   The PR has been rebased and forced pushed.  I am just waiting for the build bots to finish.